fonts/VERSION

```diff
@@ -0,0 +1 @@
+1615612969
```

Comment:
This file is to produce reproducible builds from pull requests (since we use squash and merge, the commit date may change)